packages/admin-ui-codemod/src/__tests__/radius-tokens-review.test.js

```diff
@@ -0,0 +1,96 @@
+const defineInlineTest = require('jscodeshift/dist/testUtils').defineInlineTest
+const borderRadiusTokenTransform = require('../radius-tokens-review')
+
+describe('Other token interactions', () => {
+  defineInlineTest(
+    borderRadiusTokenTransform,
+    {},
+    "<div csx={{color: '$blue'}} />",
+    "<div csx={{color: '$blue'}} />",
+    'preserves non-border tokens'
+  )
+
+  defineInlineTest(
+    borderRadiusTokenTransform,
+    {},
+    "<div csx={{color: '#000002', borderRadius: '10px'}} />",
+    "<div csx={{color: '#000002', borderRadius: '10px'}} />",
+    'preserves raw values'
+  )
+
+  defineInlineTest(
+    borderRadiusTokenTransform,
+    {},
+    "const styles = {...other, borderRadius: '$default'}; <div csx={styles} />",
+    "const styles = {...other, borderRadius: '$border-radius-base'}; <div csx={styles} />",
+    'handles spread operator'
+  )
+})
+
+describe('Responsive tokens', () => {
+  defineInlineTest(
+    borderRadiusTokenTransform,
+    {},
+    "export const baseline = style({ borderRadius: {mobile: '$flat', tablet: '$circle'} })",
+    "export const baseline = style({ borderRadius: {mobile: '$border-radius-none', tablet: '$border-radius-pill'} })",
+    'handles responsive token'
+  )
+
+  defineInlineTest(
+    borderRadiusTokenTransform,
+    {},
+    "export const baseline = style({ borderRadius: {mobile: condition ? 'default' : 'pill', tablet: 'default'} })",
+    "export const baseline = style({ borderRadius: {mobile: condition ? '$border-radius-base' : '100px', tablet: '$border-radius-base'} })",
+    'handles responsive conditional tokens'
+  )
+})
+
+describe('Token transform', () => {
+  defineInlineTest(
+    borderRadiusTokenTransform,
+    {},
+    "<Button csx={{ borderRadius: '$default' }} />",
+    "<Button csx={{ borderRadius: '$border-radius-base' }} />",
+    'transforms with single prop'
+  )
```

Comment:
Other scenario is the `default` without the `$`, I think it's missing a test to it
```
  defineInlineTest(
    borderRadiusTokenTransform,
    {},
    "<Button csx={{ borderRadius: 'default' }} />",
    "<Button csx={{ borderRadius: '$border-radius-base' }} />",
    'transforms with single prop'
  )
```

Reply:
Shouldn't it be "base" instead of "border-radius-base"?

Reply:
The test below is dedicated to that